ci/backends-markers.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
```

Comment:
can you give some comments on usage here (I know this is called from the other scripts), but nice to have